infomaniak-build-tools/windows/Readme.md

```diff
@@ -341,19 +341,19 @@ Open Visual Studio 2019 and select "Open local folder", then choose "F:\Projects
     ```
    You may need to adjust paths based on your installation.
 
-Save (CTRL + S). CMake will automatically run in the output window; ensure no errors occur.
+Save (CTRL + S). CMake will automatically run in the output window; check no errors occur.
 
-### Sync-exclude.lst
-copy and rename "F:\Projects\desktop-kDrive\sync-exclude-win.lst" to "output_path\bin\sync-exclude.lst".
+### The file `Sync-exclude.lst`
```

Comment:
```suggestion
### The file `sync-exclude.lst`
```

Reply:
I thought the installation of this files was done automatically by `CMake`.
@MaximePouce Am I right? 

If you encountered a problem with the installation of `sync-exclude.lst`, this might imply that we need to re-work some `CMakeList.txt` files.

Reply:
Agree with this remark, the copy is already made by CMake.

Reply:
I Modified the installation process, the CMake now copy the file in the right folder.